### ISSUE TITLE
Refine Evidence Description UI

### ISSUE THREAD

**Issue by notowen333** [privacy-tech-lab/privacy-pioneer#330]
As discussed in PR #324, the evidence description UI should be clarified. Specifically, I think that "Data Snippet" should be switched to "HTTP Context". Then, we would remove the "Context below 👇🏻"  point down hand. I'm wanting to keep the point down emoji and move it elsewhere, but we can also move on. 

**So, ideas for what to rename "Data Snippet" to, and where this hand can be moved (or if you think it should just be removed) are welcome.**

**Comment by SebastianZimmeck** [comment]
@notowen333 [provided some explanation](https://github.com/privacy-tech-lab/privacy-pioneer/pull/324#issuecomment-917838881) already.

**Comment by SebastianZimmeck** [comment]
We could go with your suggestion, @notowen333, i.e., "HTTP Context" instead of "Data Snippet." Another option is "Data Context."

**Comment by notowen333** [comment]
<img width="356" alt="Screen Shot 2021-09-15 at 3 38 56 PM" src="https://user-images.githubusercontent.com/51685858/133498500-01745cb9-d034-4e51-a315-7adb8498c21f.png">

@SebastianZimmeck What do you think of this?

## In this iteration
1. Switched out "Data Snippet" for "Data Context"
2. Replaced down hand with sideways hand pointing to a webpage (that we select for a particular evidence type) link

**Comment by SebastianZimmeck** [comment]
I like it! 

The only remaining comment I have is that the first two sentences: "Your fine location ... . We found ... ." are a bit wordy. Maybe, the reason for this sentence structure is that it makes sense for other privacy practices. But if that is not the case (and there are no other considerations that favor this structure either), I would make it more succinct. Maybe, just one sentence "This HTTP request contained your Fine Location (latitude and longitude): 72.663200.

On a related note, is there another such label with the latitude, 41.55... ? That would strike me as a bit formalistic if we flagged this request twice; once for latitude and once for longitude. Can we highlight both coordinates here or is that too dificult?

**Comment by notowen333** [comment]
>The only remaining comment I have is that the first two sentences: "Your fine location ... . We found ... ." are a bit wordy. Maybe, the reason for this sentence structure is that it makes sense for other privacy practices. But if that is not the case (and there are no other considerations that favor this structure either), I would make it more succinct. Maybe, just one sentence "This HTTP request contained your Fine Location (latitude and longitude): 72.663200.

This issue will be addressed in #331.

>On a related note, is there another such label with the latitude, 41.55... ? That would strike me as a bit formalistic if we flagged this request twice; once for latitude and once for longitude. Can we highlight both coordinates here or is that too difficult?

We discussed this today. It would take more work than expected and we decided it is not essential. If we decide to do it, I'll go ahead and open up an issue.

**Comment by notowen333** [comment]
Otherwise, I will make some final changes and then merge the above commits into main. #331 will iron out the actual text/links that are displayed.
